app/frontend/Calendar/EventLine.svelte

```diff
@@ -11,11 +11,12 @@
   import type { Event } from "../../logic/Calendar/Event";
   import { calendarMustangApp } from "./CalendarMustangApp";
   import { selectedEvent } from "./selected";
+  import { t } from "svelte-i18n-lingui";
 
   export let event: Event;
 
   $: startTime = event.startTime.toLocaleString(navigator.language, { hour: "2-digit", minute: "2-digit" });
-  $: eventAsText = `${startTime} – ${Math.ceil((event.endTime.getTime() - event.startTime.getTime()) / 1000 / 60)} min
+  $: eventAsText = `${startTime} – ${Math.ceil((event.endTime.getTime() - event.startTime.getTime()) / 1000 / 60)} ${$t({message: "min", comment: "Minutes Abbreviation"})}
```

Comment:
Please put the number as placeholder inside the string, so that plural forms can work properly.

Reply:
Correction: Please keep `startTime` outside the string, but put the duration number inside the translated "min" string - for plural form.